cernopendata/modules/theme/assets/semantic-ui/js/search/components/DefaultListItem.js

```diff
@@ -54,15 +54,15 @@ const CODefaultListItem = ({ result: { metadata, id }, index }) => {
             metadata.type.secondary.map((subtype) => (
               <Label
                 key={subtype}
-                className="badge badge-subtype"
+                className="tiny teal"
                 href={`/search?f=subtype:${subtype}`}
               >
                 {subtype}
               </Label>
```

Comment:
I wonder whether we should extract all these labels to a separate component and thus avoid all the repetition we have, what do you think?
So we end up with something like `<ListItemLabels metadata={metadata} />`

I know it's not part of this task, just speaking my mind here.